docker/docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: '3.3'
+
+services:
+  db:
+    image: mariadb
+    restart: always
+    environment:
+      - MARIADB_RANDOM_ROOT_PASSWORD=yes
+      - MARIADB_USER=tavern
+      - MARIADB_PASSWORD=plodder09*arouse
```

Comment:
We shouldn't commit a default password, let's use docker secrets instead if possible: https://docs.docker.com/compose/use-secrets/#advanced

Reply:
Oop this was accidental